src/main/java/duke/Event.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends duke.Task {
```

Comment:
Would it be better to change `extends duke.Task` to `extends Task`, since both classes are in the same package?